### ISSUE TITLE
ktfmt crashes when '->' is present (but no parameters) in lambda

### ISSUE THREAD

**Issue by bethcutler** [facebookincubator/ktfmt#254]
This code crashes the formatter:
```kotlin 
   check(foo > 1, { -> "foo must be legal" })
```

stack trace:
```
ktfmt: Exception in thread "main" com.google.googlejavaformat.FormattingError: 4:29: error: did not generate token "->"

        at com.google.googlejavaformat.OpsBuilder.sync(OpsBuilder.java:231)
        at com.facebook.ktfmt.KotlinInputAstVisitorBase.sync(KotlinInputAstVisitorBase.kt:2278)
        at com.facebook.ktfmt.KotlinInputAstVisitorBase.visitStringTemplateExpression(KotlinInputAstVisitorBase.kt:1828)
        at org.jetbrains.kotlin.psi.KtVisitorVoid.visitStringTemplateExpression(KtVisitorVoid.java:959)
        at org.jetbrains.kotlin.psi.KtVisitorVoid.visitStringTemplateExpression(KtVisitorVoid.java:21)
        at org.jetbrains.kotlin.psi.KtStringTemplateExpression.accept(KtStringTemplateExpression.java:47)
        at org.jetbrains.kotlin.psi.KtElementImplStub.accept(KtElementImplStub.java:60)
        at com.facebook.ktfmt.KotlinInputAstVisitorBase.visitLambdaExpression(KotlinInputAstVisitorBase.kt:834)
        at org.jetbrains.kotlin.psi.KtVisitorVoid.visitLambdaExpression(KtVisitorVoid.java:791)
        at org.jetbrains.kotlin.psi.KtVisitorVoid.visitLambdaExpression(KtVisitorVoid.java:21)
...
```

**Comment by JavierSegoviaCordoba** [comment]
What is the reason to use that arrow?

**Comment by hick209** [comment]
I don't think there's a reason to use it @JavierSegoviaCordoba, but I would be nice not to crash ktfmt in case code contains it, since it is a valid syntax.

@davidtorosyan is working on the fix

**Comment by davidtorosyan** [comment]
Fixed with 187551e
